assets/chezmoi.io/docs/reference/command-line-flags/common.md

```diff
@@ -6,10 +6,21 @@ The following flags apply to multiple commands where they are relevant.
 
 Set the output format.
 
+## `-h`, `--help`
+
+Print help.
+
 ## `-i`, `--include` *types*
 
-Include target state entries of type *types*. *types* is a comma-separated list
-of types:
+Include target state entries of type *types*.
```

Comment:
I see that you have split this so that `### Available types` presents a link target `#available-types` for use with `--exclude`, but I don't think that this improves the readability of this section.

*If* we were to consider this particular change, I think that it should instead be written as:

```markdown
Include target state entries of [*types*](#available-include-exclude-types), provided as a comma separated list.

!!! example

    `--include=files,scripts` specifies all files and scripts
```

The table would be moved to the *bottom* of this page and relabeled as `## Available Include/Exclude Types` and the `--exclude` documentation would be updated as well.

I’d need to see it to know whether it's a good change or not.

Reply:
It can be moved to the bottom or even to the new page. I don't think it matters that much. But having this list linked from other places is very useful.